### DISCUSSION TITLE
means and std

### DISCUSSION THREAD

**Discussion by victormrsilva** [autogluon/autogluon#1018] [Q&A] [answered]
Hi!

I didn't understand bagging-folds. It's the same as a repeated k-folds? How can I get the mean value for my metric and the std? And if I can't get the mean and std directly, can I get the metric value for each fold?

Thanks!

**Comment by Innixma** [comment, marked as answer]
Mean is already provided as the prediction probability of the bagged ensemble. Per-fold metric scores/predictions and std is not yet available, and is being tracked in https://github.com/awslabs/autogluon/issues/1021
